packages/react-native/src/utils/useLatestRef.ts

```diff
@@ -0,0 +1,15 @@
+import {useRef} from 'react';
+import useIsomorphicLayoutEffect from 'use-isomorphic-layout-effect';
+
+/**
+ * @see https://epicreact.dev/the-latest-ref-pattern-in-react/
+ */
+export function useLatestRef<Value>(value: Value) {
+  const ref = useRef(value);
+
+  useIsomorphicLayoutEffect(() => {
+    ref.current = value;
+  });
+
```

Comment:
React Native should always `useLayoutEffect`, no need for this check (which will actually break it, I think).

Reply:
I think the check is specifically designed not to break if `window` doesn't exist, but good catch.